scene/gui/code_edit.cpp

```diff
@@ -1000,13 +1000,42 @@ void CodeEdit::convert_indent(int p_from_line, int p_to_line) {
 	const char32_t from_indent_char = indent_using_spaces ? '\t' : ' ';
 	int size_diff = indent_using_spaces ? indent_size - 1 : -(indent_size - 1);
 	bool changed_indentation = false;
+	// 0: not in multi-line string region; 1: entered; -1 exited;
+	int enter_multiline_string_region = 0;
```

Comment:
There is a function `is_in_string()` that uses the delimiter system, it should be used instead.
Also a test in `test_code_edit.h` would be helpful.

Reply:
> There is a function `is_in_string()` that uses the delimiter system, it should be used instead. Also a test in `test_code_edit.h` would be helpful.

I've changed the implementation, and added a test.